app/experimenter/nimbus-ui/src/components/AppLayoutWithExperiment/index.tsx

```diff
@@ -18,7 +18,7 @@ import PageLoading from "../PageLoading";
 
 type AppLayoutWithExperimentChildrenProps = {
   experiment: getExperiment_experimentBySlug;
-  refetch: () => void;
+  refetch: () => Promise<unknown>;
   analysis?: AnalysisData;
 };
```

Comment:
Not sure where else we're using `refetch` - this change doesn't seem to have broken any other tests, but we might need to update this type elsewhere eventually. Technically the refetch from Apollo *is* an async function and not `() => void`, but we didn't have that plumbed through for use in `useChangeOperationMutation`